### PR TITLE
Header navigation

### PR DESCRIPTION
**Pull request by cristinakellyt**
In this commit I did the header and add new sass directories and files as it was needed.

Note: it is not responsive yet. This it will be made once we finish the home page.

### CODE REVIEW

File: sass/abstracts/_variables.scss [hunk 1]
```diff
@@ -0,0 +1,32 @@
+//COLORS
+
+$color-primary: #0e9900; //ok
```
Comment (joseigor) [inline sass/abstracts/_variables.scss:3]: what are these `//ok` that you have placed in some lines?

Comment (joseigor) [reply sass/abstracts/_variables.scss:3]: if _home.scss is not used then there is not need to be present in this pull request. 
Tip: only add files that is needed or is used by the project. Even if you think this file will be used add it only anly proper type arrives.

Comment (cristinakellyt) [reply sass/abstracts/_variables.scss]: I fixed all the errors. Thanks for the feedback

**Review by joseigor — APPROVED** [review]
Good work 🚀

**Pull request merged**